pages/home/home.html

```diff
@@ -0,0 +1,255 @@
+          </p>
+          <md-card-actions layout-xs="column" layout-align="center center">
+              <md-button class="black-text md-raised md-no-ink white-background gold-border-button md-raised">
+                Learn More
+              </md-button>
+            </md-card-actions>
+        </md-content>
+        <img ng-src="/static/images/home_get_involved.png" class="home-page-get-involved-image">
+      </md-card-content>
+    </md-card>
+
+    <md-card layout-fill class="vertical-padding gray-background">
+      <md-card-content class="horizontal-padding" layout-gt-md="row" layout-xs="column">
+        <img ng-src="/static/images/home_partner.png" class="home-page-partner-image">
+        <md-content layout="column" class="black-text gray-background horizontal-padding" layout-align="center none">
+          <h3>Partner with us</h3>
+          <p>
+            We are looking for organizations that want to join us on our journey to transform how students learn online.
+          </p>
+          <p>
+            With your help, we aim to improve our platform by conducting randomly controlled trials, fund a dedicated team of lesson creators, and manage the costs of scaling the Oppia project to serve learners across the world.
+          </p>
+          <md-card-actions layout-xs="column" layout-align="center center">
+              <md-button class="black-text md-raised md-no-ink white-background gold-border-button md-raised">
+                Learn More
+              </md-button>
+            </md-card-actions>
+        </md-content>
+      </md-card-content>
+    </md-card>
+
+    <md-card layout-fill class="vertical-padding white-background">
+      <md-card-content layout="row" layout-align="center" >
+        <md-content layout="column" class="black-text white-background donation-text-box" layout-align="center center">
+          <center>
+            <h3>Make a donation</h3>
+          </center>
+          <center>
+            <p>
+              By making a donation to The Oppia Foundation, you are helping to change millions of students’ lives. Together, we can revolutionize the way students around the world use the internet to learn.
+            </p>
+          </center>
+          <md-card-actions layout="row" layout-align="center center" >  
+            <div class="button-horizontal-padding">
+              <md-button class="black-text md-raised md-no-ink white-background gold-border-button md-raised">
+                Credit Card
+              </md-button>
+            </div>
+            <div class="button-horizontal-padding">
+              <md-button class="black-text md-raised md-no-ink white-background gold-border-button md-raised">
+                  Paypal
+              </md-button>
+            <div>
+          </md-card-actions>
+        </md-content>
+      </md-card-content>
+    </md-card>
+</section>
+
+<style>
+
+  .white-background {
```

Comment:
Start every oppia-related CSS class in this file with .home-page- ; follow a similar convention for other pages.

That also helps us differentiate from CSS local to this page vs general CSS or CSS from third-parties.

Reply:
All css classes in home.html should now be exclusive to home page.
Moving common css classes to stylesheet.css.